airbyte-scheduler/app/src/test/java/io/airbyte/scheduler/app/JobSchedulerTest.java

```diff
@@ -79,6 +79,7 @@
         .withConnectionId(connectionId)
         .withName("presto to hudi")
         .withNamespaceDefinition(NamespaceDefinitionType.SOURCE)
+        .withNamespaceFormat(null)
```

Comment:
technically we don't need to set this field right?

Reply:
correct